app/views/shared/check_answers/_client_details.html.erb

```diff
@@ -1,7 +1,7 @@
 <dl class="govuk-summary-list govuk-!-margin-bottom-9">
   <%= check_answer_link(
         name: :first_name,
-        url: providers_legal_aid_application_applicant_details_path(anchor: :first_name),
+        url: providers_legal_aid_application_applicant_details_path(@legal_aid_application, anchor: :first_name),
```

Comment:
Why do we suddenly need to update this and pass in the `@legal_aid_application`?

Reply:
because rob was relying on @legal_aid_application being understood somehow, and it doesn't work when you have parameters added to the URL (locale=en).  Actually, I think it's better to explicitly give the application id rather than rely on magic anyway.